CHANGELOG.md

```diff
@@ -2,6 +2,25 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.9.0] - 2015-12-21
+### Added
+- Add package.json for npm support
```

Comment:
Well, we're using it on Federalist but we run the build task on our side and then we move that CSS around. See [scripts here](https://github.com/18F/federalist/blob/master/package.json#L86)

I'm curious, what would you expect if you were to `import 'wds'`?